lib/jetpants/host.rb

```diff
@@ -214,6 +214,19 @@ def fast_copy_chain(base_dir, targets, options={})
       else
         output "Compression disabled -- no compression method specified in Jetpants config file"
       end
+
+      should_encrypt = false
+      targets.each do |t|
+        should_encrypt = should_encrypt || should_encrypt_with?(t)
+      end
+
+      if Jetpants.encrypt_with && Jetpants.decrypt_with && should_encrypt
```

Comment:
If should_encrypt is true, but an encrypt_with/decrypt_with has not been provided, is it desirable to silently degrade the crypto?